README.md

```diff
@@ -103,26 +103,40 @@ you'll get an error. You can fix this by either logging in (via `docker login`)
 
 Building
 -
-To build the manual you will need go-md2man.
+To build the `skopeo` binary you need at least Go 1.5 because it uses the latest `GO15VENDOREXPERIMENT` flag.
+
+### Building in a container
+If your system has a `docker` command and can run Linux containers,
 ```sh
-$ sudo apt-get install go-md2man
+$ make binary # Or (make all) to also build documentation, see below.
 ```
-To build the `skopeo` binary you need at least Go 1.5 because it uses the latest `GO15VENDOREXPERIMENT` flag. Also, make sure to clone the repository in your `GOPATH` - otherwise compilation fails.  
+(Note that the created executable is a Linux executable, and depends on dynamic libraries which may only be available only in a container of a similar Linux distribution.)
+
+### Building without a container
+Install the necessary dependencies:
 ```sh
-$ git clone https://github.com/projectatomic/skopeo $GOPATH/src/github.com/projectatomic/skopeo
-$ cd $GOPATH/src/github.com/projectatomic/skopeo && make all
+Fedora$ dnf install gpgme-devel libassuan-devel
+macOS$ brew install gpgme
```

Comment:
Yes, that flag should work as well. Do you think it is useful enough to list it in here as _another_ option, instead of just telling the users to install `gpgme`? I’m worried that the more options and variants are in the README, the more users are likely to just give up.